_posts/2018-08-02-Rust-1.28.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "Announcing Rust 1.28"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.28.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.28.0 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.28.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1280-2018-08-02
+
+## What's in 1.28.0 stable
+
+### Global Allocators
+
+1.28.0 adds the `#[global_allocator]` attribute, which allows Rust programs to
+set their allocator, as well as define new allocators by implementing the
+[`GlobalAlloc`] trait.
+
+By default, Rust uses jemalloc on Linux systems, but this may not be ideal for
+specific use cases. The standard library provides a method for switching to the
+system allocator:
+
+```rust
+use std::alloc::System;
+
+#[global_allocator]
+static GLOBAL: System = System;
+
+fn main() {
+    let mut v = Vec::new();
+    // This will allocate memory using the system allocator.
+    v.push(1);
+}
+```
+
+However, sometimes you want to define a custom allocator for a given application
+domain. This is also relatively easy to do by implementing the `GlobalAlloc`
+trait. You can read more about how to do this in the [documentation].
+
+[`GlobalAlloc`]: https://doc.rust-lang.org/1.28.0/std/alloc/trait.GlobalAlloc.html
+[documentation]: https://doc.rust-lang.org/1.28.0/std/alloc/trait.GlobalAlloc.html
+
+### Library stabilizations
+
+Several new APIs were stabilized this release:
+
+- [`Iterator::step_by`]
```

Comment:
we usually try to 

1. group similar apis in a single bullet
2. leave a short description of what the api does

writing this is the worst part of writing these blog posts :)